scripts/barclamp_mgmt_lib.rb

```diff
@@ -260,6 +260,8 @@ def generate_assets_manifest
 
   manifests = Pathname.new(CROWBAR_PATH).join("barclamps", "manifests")
 
+  manifests.mkdir unless manifests.exist?
```

Comment:
I can tell you it isn't used for anything else -- it's only a hack used for rpm packages...

I don't mind creating the directory that much. It's just not what I would have done -- and the small thread we have here is just because we were all confused about why it's needed (while it's not, it's just one of the two simple ways to fix it).